framework/source/class/qx/ui/table/pane/Scroller.js

```diff
@@ -891,7 +891,8 @@ qx.Class.define("qx.ui.table.pane.Scroller",
       var rowHeight = this.getTable().getRowHeight();
       var delta = e.getData() - e.getOldData();
       if ((Math.abs(delta) > 1) && (Math.abs(delta) < rowHeight)) {
-        delta = e.getOldData() + Math.sign(delta) * rowHeight;
+          delta = (delta < 0) ? e.getOldData() - rowHeight
```

Comment:
This looks like it inadvertently changed indentation, or has an embedded tab vs spaces?

Reply:
Oops, fixed.